app/views/staff/shared/_links.html.erb

```diff
@@ -21,7 +21,9 @@
 
   <%- if devise_mapping.omniauthable? %>
     <%- resource_class.omniauth_providers.each do |provider| %>
-      <%= govuk_link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider), method: :post %><br />
+      <% if FeatureFlag.active?(:identity_auth_service) %>
+        <%= govuk_button_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider), method: :post %><br />
+      <% end %>
     <% end %>
   <% end %>
```

Comment:
Should the feature flag go around the whole `omniauthable?` section? I guess it doesn't make much difference.